cub/cub/iterator/cache_modified_input_iterator.cuh

```diff
@@ -235,4 +236,27 @@ public:
 #endif // !_CCCL_COMPILER(NVRTC)
 };
 
+namespace detail
+{
+template <CacheLoadModifier LoadModifier, typename Iterator>
+_CCCL_DEVICE _CCCL_FORCEINLINE auto try_make_cache_modified_iterator(Iterator it)
+{
+  if constexpr (::cuda::std::contiguous_iterator<Iterator>)
+  {
+    using value_type = it_value_t<Iterator>;
+    using size_type  = it_difference_t<Iterator>;
+    return cub::CacheModifiedInputIterator<LoadModifier, value_type, size_type>{
```

Comment:
Question: We never use any other modifier than `LoadModifier`, can we just drop it as a template argument to that function?

Reply:
Oh, I totally tripped on the fact that we only ever passed `LOAD_MODIFIER`, but that is not an enum but a static variable in the policy